### ISSUE TITLE
Have a way to configure different domain for refreshing token and accessing login page

### ISSUE THREAD

**Issue by Will956** [auth0/express-openid-connect#554]
### Checklist

- [X] I have looked into the [Readme](https://github.com/auth0/express-openid-connect#readme), [Examples](https://github.com/auth0/express-openid-connect/blob/master/EXAMPLES.md), and [FAQ](https://github.com/auth0/express-openid-connect/blob/master/FAQ.md) and have not found a suitable solution or answer.
- [X] I have looked into the [API documentation](https://auth0.github.io/express-openid-connect/) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/express-openid-connect/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Describe the problem you'd like to have solved

Hello 👋 

We have some intermittent timeout issues when refreshing access tokens possibly related to our custom domain handled with Cloudfront.

To verify that, we would like to use our tenant.auth0.com URL when refreshing the token but keep our custom domain when going to the login page.

### Describe the ideal solution

As far as I understand, we can't configure the SDK to have a different domain for refreshing the token and redirecting users to the login page.

We could develop the refresh token login on our own but that will make us desync from this SDK.

Do you see any other way to achieve that?

Thanks!

### Alternatives and current workarounds

Code our own logic for the refresh token.

**Comment by frederikprijck** [comment]
I do not think this is something you should do because of reasons explained here: https://auth0.com/docs/customize/custom-domains#custom-domains-and-token-requests

TLDR: you will end up with different `iss` claim values for tokens issues on initial login and those issued as part of the refresh token grant, which will most likely open up for other issues and unexpected behavior.
